src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.ts

```diff
@@ -287,18 +289,18 @@ export class InstructorSessionResultPageComponent extends InstructorCommentsComp
       feedbackSessionName: this.session.feedbackSessionName,
       intent: Intent.INSTRUCTOR_RESULT,
     })
-    .subscribe((resp: SessionResults) => {
-      if (resp.questions.length) {
-        const responses: QuestionOutput = resp.questions[0];
-        this.questionsModel[questionId].responses = responses.allResponses;
-        this.questionsModel[questionId].statistics = responses.questionStatistics;
-        this.questionsModel[questionId].hasPopulated = true;
-
-        this.preprocessComments(responses.allResponses);
-      }
-    }, (resp: ErrorMessageOutput) => {
-      this.statusMessageService.showErrorToast(resp.error.message);
-    });
+        .subscribe((resp: SessionResults) => {
+          if (resp.questions.length) {
+            const responses: QuestionOutput = resp.questions[0];
+            this.questionsModel[questionId].responses = responses.allResponses;
+            this.questionsModel[questionId].statistics = responses.questionStatistics;
+            this.questionsModel[questionId].hasPopulated = true;
+
+            this.preprocessComments(responses.allResponses);
+          }
+        }, (resp: ErrorMessageOutput) => {
+          this.statusMessageService.showErrorToast(resp.error.message);
+        });
```

Comment:
various parts of your codes have the spacing changed. I can't merge as merging it is very likely to incur CI failure in my PR, which would demand a commit to fix (either me or you). Please do it here.